ros2_ws/src/rcdt_sensors/nodes/combine_camera_topics_node.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import rclpy
+from rclpy.node import Node
+from sensor_msgs.msg import Image
+from realsense2_camera_msgs.msg import RGBD
+
+
+class JoyToTwistNode(Node):
+    def __init__(self) -> bool:
+        super().__init__("combine_camera_topics")
+        self.declare_parameter("rgb_topic", "/camera/color/image_raw")
+        self.declare_parameter("d_topic", "/camera/depth/image_rect_raw")
+        self.declare_parameter("rgbd_topic", "/camera/rgbd")
+        self.declare_parameter("rate", 30)
+
+        rgb_topic = self.get_parameter("rgb_topic").get_parameter_value().string_value
+        d_topic = self.get_parameter("d_topic").get_parameter_value().string_value
+        rgbd_topic = self.get_parameter("rgbd_topic").get_parameter_value().string_value
+        rate = self.get_parameter("rate").get_parameter_value().integer_value
+
+        self.create_subscription(Image, rgb_topic, self.update_rgb, 10)
+        self.create_subscription(Image, d_topic, self.update_d, 10)
+        self.update = False
```

Comment:
Het geheel met self.update zit niet helemaal lekker. Als er bijvoorbeeld helemaal geen dieptebeelden worden gepubliceerd dan wordt er alsnog een half RGBD-message gepubliceerd, en als er helemaal geen rgb-beelden worden gepubliceerd dan wordt er nooit een RGBD-message gepubliceerd (opzich kan dit zijn wat we willen).

Voor nu is het misschien het meest eenvoudige om gewoon telkens het meest recente beeld te publiceren. Dit is niet ideaal, maar er moet toch nog gekeken worden naar het synchroniceren van de topics en het alignen